include/revng/Model/CollectAccesses.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
```

Comment:
1. This file does not seem to be included by anyone.
2. Files should have the same name as the "main" thing in a class.
3. This file only contains stuff in the `detail` namespace.